### DISCUSSION TITLE
Different results Hunt vs Kibana while using same query

### DISCUSSION THREAD

**Discussion by mnasec** [Security-Onion-Solutions/securityonion#7535] [Unsupported Versions] [answered]
Using SO 2.3.110 in distributed environment.
When using the same query in Kibana and Hunt, i got different results in Hunt, it seems that the OR connection with same fields does not work correctly:

Kibana:
`destination.port:1389 or destination.port:2420`
Result 3 matches:
![image](https://user-images.githubusercontent.com/55296987/158034670-ac9ab08b-335b-406e-bde7-ba07c5fe3980.png)

Hunt:
`destination.port:1389 or destination.port:2420`
Result 1 match:
![image](https://user-images.githubusercontent.com/55296987/158034717-fda9eae8-42a3-4c07-9946-04fcb9e3082a.png)

Hunt (changed query arguments): 
`destination.port: 2420 or destination.port 1389`
Result 2 matches:
![image](https://user-images.githubusercontent.com/55296987/158034774-d267335d-ed77-4cd1-ae42-aa219f3d3c5a.png)

I got the same result while using captial OR.

Question: Bug or do i use the wrong query format in Hunt?

**Comment by dougburks** [comment, marked as answer]
This appears to be due to a recent change and we should be able to improve this for the next release. In the meantime, can you try wrapping your Hunt query in parentheses and verify that it works for you?

For example:
```
(destination.port: 2420 OR destination.port 1389)
```

**Reply by mnasec** [reply]
yepp, this workaround works - thanks a lot!

**Reply by dougburks** [reply]
Thanks for confirming! I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/7540 to fix this in the next release.
